Server/src/Config/Config.java

```diff
@@ -26,7 +27,7 @@ private Config() {
             AppDirs appDirs = AppDirsFactory.getInstance();
             String configDirPath = appDirs.getUserConfigDir("cardboard", "1.0", "projectcardboard", true);
             String configFileName = "/config.properties";
```

Comment:
better would probably be "configDirPath + "/" + configFileName

even better would be to use a Java library that concatenates paths for you.  (code like the about may break on other operating systems, for example).